lib/translate-typeform/translate-fields.js

```diff
@@ -191,8 +191,28 @@ const translateShare = (data, ref) => {
   return response
 }
 
-const translateWebview = (data, ref) => {
-
+const translateWebview = data => {
+  const { url } = data.md
+  const response = {
+    "attachment":{
+      "type":"template",
+      "payload":{
+        "template_type":"button",
+        "text": data.title,
+        "buttons":[
+          {
+            "type":"web_url",
+            "url": url,
+            "title":"View website",
```

Comment:
We should let the user decide the button text! Get it from the metadata, like in translateShare.